pkg/api/v1/state.go

```diff
@@ -65,7 +65,7 @@ func findDashboardTab(cfg *cachedConfig, dashboardInput string, tabInput string)
 
 // GroupGrid fetch tab group name grid info (columns, rows, ..etc)
 func (s Server) GroupGrid(ctx context.Context, configPath *gcs.Path, groupName string) (*statepb.Grid, error) {
```

Comment:
nit: I think GroupGrid() shouldn't be referenced anymore with the lines below removed, so it should be safe to delete entirely. (It's good to confirm that's the case though).

Reply:
+1 to this. This fetches the data from the updater rather than tabulator